### ISSUE TITLE
Intellij search dialog invisible in full screen mode

### ISSUE THREAD

**Issue by s1ck** [mate-desktop/marco#520]
#### Expected behaviour

When using Intellij in fullscreen or presentation mode, pressing double shift opens a search dialog.

#### Actual behaviour

When using Intellij in fullscreen or presentation mode, pressing double shift activates the search dialog but does not display the dialog itself.

#### Steps to reproduce the behaviour

* Open any project in Intellij (I'm using Java and Sala projects)
* Change to full screen (View -> Enter full screen)
* Press double shift (or use Strg + Shift + N)

#### MATE general version

* `MATE 1.20.1`
* `MATE 1.20.4`

#### Package version

* `1.20.1-2ubuntu1`
* `1.20.4-2`

#### Linux Distribution

* Release 18.04.2 LTS 64-bit, Kernel Linux 4.15.0-51-generic x86_64
* Release 19.04 64-bit, Kernel Linux 5.0.0-16-generic x86_64

**Comment by s1ck** [comment]
I see the following debug output when pressing double shift in both non-full screen and full screen, maybe it helps.
```
Window manager warning: Window 0x2600a77 (win16) sets an MWM hint indicating it isn't resizable, but sets min size 1 x 1 and max size 2147483647 x 2147483647; this doesn't make much sense.
```

**Comment by lukefromdc** [comment]
Can you try this in metacity, compiz, or another window manager and see what happens? The bug could be in the program itself and not the window manager.

**Comment by raveit65** [comment]
Or better with another graphic driver?

**Comment by s1ck** [comment]
I can certainly test with another window manager. However, the problem does not occur with plain Ubuntu (Gnome 3), PopOS (modified Gnome 3) and also not on MacOS (verified by a colleague). I will also open an issue on Jetbrains for Intellij.
@raveit65 I think the driver can be ruled out as I reproduced the issue on different laptop and desktop machines, both with Intel and nvidia GPUs.

**Comment by raveit65** [comment]
Which drver? pour nouveau or exelent nvidia?
i915 or better modeset driver?

**Comment by s1ck** [comment]
@raveit65 I tried compiz and it now works as expected. The issue seems to be related to marco. I'm using the nvidia driver.

**Comment by s1ck** [comment]
The issue seems to be resolved in Intellij 2.1 (however, I couldn't find the corresponding fix in their changelog). I can finally work in fullscreen / distraction-free mode!
